src/cirrus/builtins/cloudformation/resources.yml

```diff
@@ -78,14 +78,12 @@ Resources:
 
   StateEventTimestreamDatabase:
     Type: AWS::Timestream::Database
-    Condition: TimestreamIsSupportedInRegion
```

Comment:
Do you also want to remove the top level `Conditions` for this defined in this file?